tox.ini

```diff
@@ -17,10 +18,12 @@ commands =
     travis: - coveralls
 
 deps =
-    django22: Django~=2.2
-    django30: Django~=3.0
-    django31: Django~=3.1
-    djangorestframework~=3.10
+    django21: Django~=2.1.0
```

Comment:
why was django 2.1 added? I don't think we should be trying to actively support old versions of django due to our limited resources.

Reply:
Sure. I will remove it.